newmiddleware/auth.js

```diff
@@ -32,11 +32,10 @@ class AuthMiddleware {
   // Replace access token with site access token if it is available
   async useSiteAccessTokenIfAvailable(req, res, next) {
     const {
-      accessToken: userAccessToken,
       params: { siteName },
     } = req
     const {
-      locals: { userId },
+      locals: { userId, accessToken: userAccessToken },
     } = res
```

Comment:
Minor nit: could we swap this to
```
const { siteName } = req.params
const { userId, accessToken: userAccessToken } = res.locals
```
?

Same for newroutes/authenticated/sites!

Reply:
good catch!! i think this helps with readability and i'd be more than happy to make this change!